### PR TITLE
Add an integration test for submit-feedback endpoint

### PR DESCRIPTION
**Pull request by sara-02**
https://github.com/openshiftio/openshift.io/issues/1740

### CODE REVIEW

File: integration-tests/features/steps/server_api.py [hunk 1]
```diff
@@ -38,3 +51,8 @@ def check_versions(context, num=0, versions='', ecosystem='', package=''):
         assert v['ecosystem'] == ecosystem
         assert v['package'] == package
         assert v['version'] in versions
+
+
+@then('I should get 400 status code')
+def check_error_code(context):
```
Comment (tisnik) [inline integration-tests/features/steps/server_api.py]: Please note that we need to document all public methods and functions.

File: integration-tests/features/steps/server_api.py [hunk 2]
```diff
@@ -18,6 +18,19 @@ def access_url_with_authorization_token(context, url):
                                     headers=authorization(context))
 
 
+@when('I access {url:S} without valid values')
+def check_submit_feedback(context, url):
+    payload = {
```
Comment (tisnik) [inline integration-tests/features/steps/server_api.py]: Please note that we need to document all public methods and functions.

File: integration-tests/features/steps/server_api.py (hunk 1 above)
Comment (tisnik) [reply integration-tests/features/steps/server_api.py]: Additionally, the prerequisity 'I should get 400 status code' is already covered by implemented test step.

Comment (tisnik) [reply integration-tests/features/steps/server_api.py]: We don't need this function ;)

**Comment by sara-02** [comment]
@tisnik Then how will the assertion work?

**Comment by sara-02** [comment]
[test]

**Comment by tisnik** [comment]
[test this]

**Comment by tisnik** [comment]
@sara-02 please rebase to get https://github.com/fabric8-analytics/fabric8-analytics-common/pull/467

File: integration-tests/features/server_api.feature [hunk 3]
```diff
@@ -37,3 +37,8 @@ Feature: Server API
     Then I should get 200 status code
     Then I should receive JSON response containing the api key
 
+  @production
+  Scenario: Check the /api/v1/submit-feedback response
+    Given System is running
+    When I access /api/v1/submit-feedback without valid values
```
Comment (tisnik) [inline integration-tests/features/server_api.feature]: Please add:
```
    When I acquire the authorization token
    Then I should get the proper authorization token
```

File: integration-tests/features/server_api.feature [hunk 4]
```diff
@@ -37,3 +37,8 @@ Feature: Server API
     Then I should get 200 status code
     Then I should receive JSON response containing the api key
 
+  @production
+  Scenario: Check the /api/v1/submit-feedback response
+    Given System is running
```
Comment (tisnik) [inline integration-tests/features/server_api.feature]: Please add:
```
    When I acquire the authorization token
    Then I should get the proper authorization token
```

**Comment by tisnik** [comment]
[test]

**Comment by sara-02** [comment]
@tisnik PTAL again, I rebased and included your changes.

**Comment by tisnik** [comment]
LGTM, thanks @sara-02

**Pull request merged**